.github/workflows/build.yml

```diff
@@ -126,8 +126,58 @@ jobs:
       - name: Run tests
         run: cargo test
 
+  build-ci-base :
+    runs-on: ubuntu-22.04
+    outputs:
+      output: ${{ steps.export_tag.outputs.image_tag }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+  
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            name=ghcr.io/cartesi/rollups-node-ci-base
+          tags: |
+            type=semver,pattern={{version}}
+            type=ref,event=branch
+            type=ref,event=pr
+  
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+  
+      - uses: depot/setup-action@v1
+      - name: Build and push docker image
+        id: docker_build
+        uses: depot/bake-action@v1
+        with:
+          files: |
+            ./docker-bake.hcl
+            ${{ steps.docker_meta.outputs.bake-file }}
+            ./docker-bake.platforms.hcl
+          targets: rollups-node-ci-base
+          push: true
```

Comment:
You should add this image to the clean-up job.
https://github.com/cartesi/rollups-node/blob/74d4381f84fb68ee9d6aa2baa9a3bff436b34b3f/.github/workflows/clean-up-images.yml#L19